### DISCUSSION TITLE
[labs/context] I have issue with provide and private properties

### DISCUSSION THREAD

**Discussion by NealRame** [lit/lit#3875] [Q&A]
In my component I want to add a context properties which is also private
```
@provide({ context: someContext })
@state()
private _someData = 0
```

I read in the documentation that it is possible however when doing so, I get that typescript compiler error :
```
error TS2345: Argument of type 'MyComponent' is not assignable to parameter of type 'ReactiveElement & Record<"_someData", number>'.
  Type 'MyComponent' is not assignable to type 'Record<"_someData", number>'.
    Property '_someData' is private in type 'MyComponent' but not in type 'Record<"_someData", number>'.

89     @provide({ context: someContext })
```

Could someone explain me if I am missing something please?

I am using `lit@2.6.1` and `@lit-labs/context@0.3.1`.

**Comment by justinfagnani** [comment]
I see the problem here. I'm going to convert this into an issue. Thanks for the report!

**Reply by AndrewJakubowicz** [reply]
For reference - Tracking issue: https://github.com/lit/lit/issues/3883

Thank you for capturing this in an issue!
